File: webhook.js

const axios = require('axios');

const webhookURL = 'https://webhook.site/5364a595-6b43-4b6e-9ba1-b11e272a140f';

const data = {
  text: 'Hello, World!'
};

axios.post(webhookURL, data)
  .then(response => {
    console.log(`Status: ${response.status}`);
    console.log('Body: ', response.data);
  })
  .catch(error => {
    console.error(`Error: ${error}`);
  });

File: webhook.py
import requests
import json

webhook_url = 'https://webhook.site/5364a595-6b43-4b6e-9ba1-b11e272a140f'
data = { 'name' : 'nighthack' ,
         'message' : 'Hello World' }

r = requests.post(webhook_url, data=json.dumps(data), headers={'Content-Type': 'application/json'})



